addons/l10n_za/data/template/account.account-za.csv

```diff
@@ -30,6 +32,8 @@
 "200090","200090","Hire Purchase","liability_current","False"
 "200100","200100","Mortgages","liability_current","False"
 "210010","210010","Company Credit Card","liability_credit_card","False"
+"211008","211008","Creditors Suspense Account","asset_current","False"
```

Comment:
I'm not sure what these debtors/creditos suspense accounts are for? could it be a duplication of the automatically created outstanding payment/ receipts accounts?
Maybe a better question for an accountant @Levizar or @tsb-odoo

Reply:
It looks indeed as a duplication of the outstanding payment/receipts accounts, except that it seems it's not only for payment/receipt.
In Belgium we have the 499000 account for that.
You can put anything for which you're not sure in it BUT at the end of the year, it **HAS TO** be completely empty.